### DISCUSSION TITLE
Identify the Domain That Started a Video Conference in Reports Analitycs

### DISCUSSION THREAD

**Discussion by genius987** [mynaparrot/plugNmeet-server#549] [Q&A]
Is it possible to see which domain initiated the video conference? For example, if I used Plugnmeet on Moodle running on the domain server1.moodleschool.com, can the reports indicate that the room was started by server1.moodleschool.com?

**Comment by jibon57** [comment]
In the analytics host name never store. You can use room id to find which one from which domain. Normally room id should be unique.
